### PR TITLE
feat(whitelist): add Layer3 to whitelisted protocols

### PR DESCRIPTION
**Pull request by yahya-ea**
### Title

feat(whitelist): add Layer3 to whitelisted protocols

---

#### What does this PR do?

This PR adds Layer3 to the list of whitelisted protocols for the Trailblazer program. Specifically, it:

Adds a Layer3 entry to the protocols.json file
Uploads the Layer3 logo (layer3.png) to the img folder

---

### Checklist

- [x] I have followed the conventional commit messages standard.
- [x] Any whitelisted contracts added are verified.

---

### Additional Notes

The Layer3 contract address added is 0x1195Cf65f83B3A5768F3C496D3A05AD6412c64B7. This is the contract for the CUBE Credentials protocol. 

---

### CODE REVIEW

**Comment by yahya-ea** [comment]
@bennettyong should be fixed

**Comment by yahya-ea** [comment]
hi @bennettyong anything else needed on our side?

File: whitelist/protocols.json [hunk 1]
```diff
@@ -861,6 +861,14 @@
       "0x7BD616192fB2B364f9d29B2026165281a5f2ff2F"
     ]
   },
+  {
+    "name": "Layer3",
+    "slug": "layer3",
+    "twitter": "@layer3xyz",
+    "logo": "layer3.png",
+    "contracts": [
+      "0x1195Cf65f83B3A5768F3C496D3A05AD6412c64B7"
```
Comment (KorbinianK) [inline whitelist/protocols.json:870]:
```suggestion
      "0x1195Cf65f83B3A5768F3C496D3A05AD6412c64B7"
      ]
```

File: whitelist/protocols.json [hunk 2]
```diff
@@ -861,6 +861,15 @@
...
+      ]
+  },
```
Comment (myronrotter) [inline whitelist/protocols.json:872]:
```suggestion
  {
    "name": "Layer3",
    "slug": "layer3",
    "twitter": "@layer3xyz",
    "logo": "layer3.png",
    "contracts": [
      "0x1195Cf65f83B3A5768F3C496D3A05AD6412c64B7"
    ]
  },
```

**Review by myronrotter — APPROVED** [review]

**Pull request merged**